pkg/route/tree.go

```diff
@@ -441,7 +439,7 @@ func (r *router) find(path string, paramsPointer *param.Params, unescape bool) (
 				}
 			}
 
-			(*paramsPointer)[index].Value = bytesconv.B2s(append(buf, val...))
```

Comment:
This change may have  some promotion for the performance of `Any` route.
Maybe add a benchmark stats in your pr description？